### PR TITLE
Html and CSS of Hello World Project

### PR DESCRIPTION
**Pull request by trace-kadenyi**
- Added header
- Added color to header

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*test.md
```
Comment (arslanbisharat) [inline .gitignore]: - [x] You added test.md in gitignore but this not how it should be. Please read https://www.atlassian.com/git/tutorials/saving-changes/gitignore#:~:text=If%20you%20want%20to%20ignore,directory%20as%20an%20ignored%20file.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (trace-kadenyi) [reply .gitignore]: Thank you for your review. It was helpful. I have amended my code.

File: test,md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
Comment (youmari) [inline test,md]: - [x] [OPTIONAL] you should remove this file as required it doesn't get ignored cause you name it `test,md` instead of `test.md`

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @TracK92 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

 ## Highlights
✔️ Github flow 
✔️ Great title and short summary 

 

- [x] [OPTIONAL] please try to add a descriptive readme file using the [template](https://github.com/microverseinc/readme-template) provided by microverse so that your project will be well documented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: test,md (hunk 2 above)
Comment (trace-kadenyi) [reply test,md]: Thank you very much for your feedback. I have made changed according to your optional suggestions.

**Pull request merged**